### PR TITLE
Zennarchi/redux tests

### PR DESCRIPTION
**Pull request by zennarchi**

### CODE REVIEW

File: client/src/store/alert/tests/actions.spec.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { showAlert, clearAlert } from '../actions';
+import * as t from '../actionTypes';
```
Comment (InnaKutsik) [inline client/src/store/alert/tests/actions.spec.ts]: types instead of t

File: client/src/store/alert/tests/reducers.spec.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import { alertReducer, initialState } from '../reducers';
+import * as t from '../actionTypes';
```
Comment (InnaKutsik) [inline client/src/store/alert/tests/reducers.spec.ts]: types instead of t

File: client/src/store/checkEmail/tests/actions.spec.ts [hunk 3]
```diff
@@ -0,0 +1,58 @@
+import configureMockStore from 'redux-mock-store';
+
+import thunk from 'redux-thunk';
+
+import { API } from '../../api';
+import * as t from '../../alert/actionTypes';
```
Comment (InnaKutsik) [inline client/src/store/checkEmail/tests/actions.spec.ts]: types instead of t

File: client/src/store/cropper/tests/actions.spec.ts [hunk 4]
```diff
@@ -0,0 +1,60 @@
+import configureMockStore from 'redux-mock-store';
+
+import thunk from 'redux-thunk';
+
+import { API } from '../../api';
+import * as t from '../actionTypes';
```
Comment (InnaKutsik) [inline client/src/store/cropper/tests/actions.spec.ts]: types instead of t

File: client/src/store/login/tests/actions.spec.ts [hunk 5]
```diff
@@ -0,0 +1,62 @@
+import configureMockStore from 'redux-mock-store';
+
+import thunk from 'redux-thunk';
+
+import { API } from '../../api';
+import * as t from '../../alert/actionTypes';
```
Comment (InnaKutsik) [inline client/src/store/login/tests/actions.spec.ts]: types instead of t

File: client/src/store/register/tests/actions.spec.ts [hunk 6]
```diff
@@ -0,0 +1,76 @@
+import configureMockStore from 'redux-mock-store';
+
+import thunk from 'redux-thunk';
+
+import { API } from '../../api';
+import * as t from '../../alert/actionTypes';
```
Comment (InnaKutsik) [inline client/src/store/register/tests/actions.spec.ts]: types instead of t

File: client/src/store/alert/tests/actions.spec.ts (hunk 1 above)
Comment (zennarchi) [reply client/src/store/alert/tests/actions.spec.ts]: Done

File: client/src/store/alert/tests/reducers.spec.ts (hunk 2 above)
Comment (zennarchi) [reply client/src/store/alert/tests/reducers.spec.ts]: Done

File: client/src/store/checkEmail/tests/actions.spec.ts (hunk 3 above)
Comment (zennarchi) [reply client/src/store/checkEmail/tests/actions.spec.ts]: Done

File: client/src/store/cropper/tests/actions.spec.ts (hunk 4 above)
Comment (zennarchi) [reply client/src/store/cropper/tests/actions.spec.ts]: Done

File: client/src/store/login/tests/actions.spec.ts (hunk 5 above)
Comment (zennarchi) [reply client/src/store/login/tests/actions.spec.ts]: Done

File: client/src/store/register/tests/actions.spec.ts (hunk 6 above)
Comment (zennarchi) [reply client/src/store/register/tests/actions.spec.ts]: Done

**Pull request merged**